src/custom/components/swap/EthFlow/helpers.ts

```diff
@@ -184,49 +184,48 @@ export type ModalTextContentProps = {
 // returns modal content: header and descriptions based on state
 export function _getModalTextContent(params: ModalTextContentProps) {
   const { wrappedSymbol, nativeSymbol, state, isExpertMode, isNative /*, wrapSubmitted, approveSubmitted */ } = params
-  // common text
-  const swapHeader = `Swap ${wrappedSymbol}`
-  const swapDescription = `Click "Swap" to submit an off-chain transaction and swap your ${wrappedSymbol}`
-  const signatureRequiredDescription = 'Transaction signature required, please check your connected wallet'
-  const transactionInProgress = 'Transaction in progress. See below for live status updates'
```

Comment:
i couldn't help to move some of this "common" messages to the SWITCH. It was not true they were common.